apps/fares/lib/fare_info.ex

```diff
@@ -264,15 +264,15 @@ defmodule Fares.FareInfo do
         mode: :ferry,
         name: :ferry_inner_harbor,
         duration: :single_trip,
-        media: [:charlie_ticket, :cash, :mticket],
+        media: [:mticket, :paper_ferry, :cash],
```

Comment:
We should update the type at `Fares.Fare` and also possibly at `Site.ContentRewriters.LiquidObjects.Fare` to support this new type of media.